cargo/resolver-pubgrub-office-hours.toml

```
@@ -12,9 +12,10 @@ Propose topics at https://rust-lang.zulipchat.com/#narrow/stream/260232-t-cargo.
 """
 location = "https://meet.jit.si/CargoOfficeHours"
 created_on = "2024-01-09T19:08:15.06Z"
-last_modified_on = "2024-01-17T16:48:03.72Z"
+last_modified_on = "2024-03-19T16:48:03.72Z"
 start = { date = "2024-01-17T13:00:00.00", timezone = "America/New_York" }
 end = { date = "2024-01-17T15:00:00.00", timezone = "America/New_York" }
 status = "confirmed"
 organizer = { name = "T-cargo", email = "cargo@rust-lang.org" }
 recurrence_rules = [ { frequency = "weekly" } ]
+exceptions = [ "2024-03-27T13:00:00.00Z", "2024-04-03T13:00:00.00Z", "2024-04-17T13:00:00.00Z", "2024-04-24T13:00:00.00Z" ]
```

These don't line up with the time as we specify the timezone to be New York, and exceptions require UTC. UTC it should be 17:00.
```suggestion
exceptions = [ "2024-03-27T17:00:00.00Z", "2024-04-03T17:00:00.00Z", "2024-04-17T17:00:00.00Z", "2024-04-24T17:00:00.00Z" ]
```
Note: I might make a PR to make this less confusing in the future